### ISSUE TITLE
Developer Docs page

### ISSUE THREAD

**Issue by sbddesign** [sbddesign/bolt12.org#33]
Extended information and examples. This goes beyond just what is shown on the "How to Integrate" part of the homepage.

**Comment by thebrandonlucas** [comment]
I wonder if, in v2, we should consider using something like [docusaurus](https://docusaurus.io/docs/playground). [Fedimint](https://fedimint.org/) uses it, and we're using it for a site redesign for [payjoin.org](https://payjoin.org). We can keep whatever pages we have now styled as we have them, and then just write whatever docs we need in markdown. It would be a big lift, so I think something that should be saved for a v2, but it could provide a very natural progression from "what is Bolt12 and how does it work?" -> "how do I use it?" -> "API".

**Comment by sbddesign** [comment]
Docusaurus or something could be nice, but is it compatible with NextJS?

Here's my current thinking: most of these these docs for the time being are going to be things like "this implementation supports it, click here to this blog to find out more".

In the future, if we need to expand this content, we can do that.

If it starts to get to be a LOT of content, then maybe we look at Docusaurus. I have also setup ContentLayer markdown stuff before, which I know works with NextJS and will turn markdown files into docs.

But lets see how many docs we actually need first.

**Comment by thebrandonlucas** [comment]
> Docusaurus or something could be nice, but is it compatible with NextJS?
It looks like it is, but perhaps [with some work](https://docusaurus.io/docs/next#nextjs). 

> If it starts to get to be a LOT of content, then maybe we look at Docusaurus. I have also setup ContentLayer markdown stuff before, which I know works with NextJS and will turn markdown files into docs.

Agreed 🤝 . Definitely something we don't need now, I just found it very convenient for structuring docs pages and it's a design people are familiar with

**Comment by sbddesign** [comment]
I am closing this because it's done and/or too vague. We have a Developer Docs page. I think we all have a general sense that "I want to see more cool stuff" on this page. However, that varies from one implementation to the next, so it would be better to get input from the various LN implementations. It should also be motivated by "what's useful?" For example, @thebrandonlucas, after we finish our mutinynet experiment, we could document some of that stuff here.

In other words, closing this issue because it's done "good enough" for now. We can open smaller, more targeted issues in future to add specific kinds of content later.
